src/main/webapp/app/my-events.jsp

```diff
@@ -1,16 +1,89 @@
-<%--
-  Created by IntelliJ IDEA.
-  User: pavel.tihomirov01
-  Date: 8/21/2018
-  Time: 16:47
-  To change this template use File | Settings | File Templates.
---%>
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 <html>
 <head>
     <title>My Events</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <script src="http://www.w3schools.com/lib/w3data.js"></script>
 </head>
-<body>
+<style>
+    ul{
+       border: solid;
+        text-align: left;
+    }
+    li {
+
+        border: thin;
+        width: auto;
+    }
+
+    #edit {
+        margin-left: 600px;
+    }
+
+    #createPoolButton {
+        margin-left: 10px;
+    }
+    #header{
+        border: dotted;
+        text:bold;
+        text-align: center;
+    }
+    a {
+        text-underline-mode: none;
+    }
```

Comment:
Styles should be kept in a CSS file to allow reusing them and keep them consistent across several pages.